packages/valory/protocols/tendermint/custom_types.py

```diff
@@ -19,16 +19,17 @@
 
 """This module contains class representations corresponding to every custom type in the protocol specification."""
 
+from enum import Enum
```

Comment:
<img width="1440" alt="Screenshot 2022-04-08 at 17 13 01" src="https://user-images.githubusercontent.com/19314578/162555597-83883744-5b9e-4d5b-a3a8-cd1c4aeb5358.png">

Reply:
Then the custom type also needs to be implemented; which you had not done.